extensions/emmet/src/util.ts

```diff
@@ -49,6 +49,40 @@ export function updateEmmetExtensionsPath(forceRefresh: boolean = false) {
 	}
 }
 
+/**
+ * Migrate old configuration(string) for extensionsPath to new type(string[])
+ * https://github.com/microsoft/vscode/issues/117517
+ */
+export function migrateEmmetExtensionsPath() {
+	// Get the detail info of emmet.extensionsPath setting
+	let config = vscode.workspace.getConfiguration().inspect('emmet.extensionsPath');
+
+	// Update Global setting if the value type is string
+	if (typeof config?.globalValue === 'string') {
+		vscode.workspace.getConfiguration().update('emmet.extensionsPath', [config.globalValue], true);
+	}
+	// Update Global setting if the value is null
+	if (config?.globalValue === null) {
```

Comment:
This can be an `else if` rather than just an `if`, because if the global value above was a string, this case doesn't occur. Same thing for the cases below.

Reply:
That make sense.